redbot/core/core_commands.py

```diff
@@ -1877,6 +1877,52 @@ class Core(commands.Cog, CoreLogic):
         """Manage the bot's commands."""
         pass
 
+    @command_manager.group(name="listdisabled", invoke_without_command=True)
+    async def list_disabled(self, ctx: commands.Context):
+        """
+        List disabled commands.
+
+        If you're the bot owner, this will show global disabled commands by default.
+        """
+        # Select the scope based on the author's privileges
+        if await ctx.bot.is_owner(ctx.author):
+            await ctx.invoke(self.list_disabled_global)
+        else:
+            await ctx.invoke(self.list_disabled_guild)
+
+    @list_disabled.command(name="global")
+    async def list_disabled_global(self, ctx: commands.Context):
+        """List disabled commands globally."""
+        disabled_list = await self.bot._config.disabled_commands()
+        if disabled_list:
+            msg = _("{num:,} {cmd} {plur} disabled globally.\n").format(
+                num=len(disabled_list),
+                cmd=_("commands") if len(disabled_list) > 1 else _("command"),
+                plur=_("are") if len(disabled_list) > 1 else _("is"),
```

Comment:
Splitting up the individual words of the message can still cause issues. It would probably best to have two separate strings (one singular and one plural) and an `if` statement to use the appropriate one.